src/main/resources/assets/js/application.js

```diff
@@ -0,0 +1,25 @@
+function getHostName(url) {
+  var match = url.match(/:\/\/(www[0-9]?\.)?(.[^/:]+)/i);
```

Comment:
just to be clear I mean 
`document.body.className = url.(s.indexOf("gov.uk") == -1) ? "open-registers" : "gov-uk";` 
would effectively do the same thing.